api/config/routes.rb

```diff
@@ -101,7 +101,14 @@
       resources :subjects
       resources :categories, except: [:create, :index]
       resources :makers
-      resources :ingestions, only: [:show, :update]
+      resources :ingestions, only: [:show, :update] do
+        member do
+          get "show_messages"
+          patch "reset"
```

Comment:
Use post, not patch to match the other similar actions we have that use these sorts of methods.